contracts/build/contracts/ADai.json

```diff
@@ -549,14 +549,14 @@
     "absolutePath": "/Users/johanthan/Code/hackathons/app/contracts/contracts/mocks/ADai.sol",
```

Comment:
I'm going to add these to the gitignore file (it is pointless commiting them every time even if we aren't deploying!)